portlist/portlist_linux.go

```diff
@@ -171,7 +198,7 @@ func parsePorts(r *bufio.Reader, fileBase string) ([]Port, error) {
 		ret = append(ret, Port{
 			Proto: proto,
 			Port:  uint16(portv),
-			inode: string(inoBuf),
+			inode: stringCache.get(inoBuf),
```

Comment:
If I'm reading this correctly, "socket:[inode_number]" strings will be put into the pool for unix local sockets, and there isn't a bound on the size of the pool. The potential number of inodes is large, if sockets are being created regularly.

Will there only ever be one socket, the one between tailscale and tailscaled?

Reply:
There will be one per listening process.

They're at least in a sync.Pool so if/when there's memory pressure the whole map will be dropped.

But I can make it better and drop the map if/when it gets double the expected size.

Reply:
PTAL

Reply:
LGTM